File: Mediator/globits-hiv-info-mediator/src/main/java/dto/TuberculosisDto.java
package dto;

import java.util.Date;
import java.util.List;

public class TuberculosisDto {
    public Date NgayChanDoan; //TB Diagnosis Date
    public List<HivDPLaoDto> listTPT;
    public List<HivDTLaoDto> listTbTreatment;
	public Date getNgayChanDoan() {
		return NgayChanDoan;
	}
	public void setNgayChanDoan(Date ngayChanDoan) {
		NgayChanDoan = ngayChanDoan;
	}
	public List<HivDPLaoDto> getListTPT() {
		return listTPT;
	}
	public void setListTPT(List<HivDPLaoDto> listTPT) {
		this.listTPT = listTPT;
	}
	public List<HivDTLaoDto> getListTbTreatment() {
		return listTbTreatment;
	}
	public void setListTbTreatment(List<HivDTLaoDto> listTbTreatment) {
		this.listTbTreatment = listTbTreatment;
	}

}

File: Adapter/globits-hiv-info-adapter/GlobitsHivInfoAdapter/Dto/TuberculosisDto.cs
using GlobitsHivInfoAdapter.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GlobitsHivInfoAdapter.Dto
{
    public class TuberculosisDto
    {
        public Nullable<DateTime> NgayChanDoan { get; set; } //TB Diagnosis Date
        public List<HIV_DPLao> listTPT;
        public List<HIV_DTLao> listTbTreatment;
    }
}
